### ISSUE TITLE
Cannot find module '@aws-sdk/client-lambda-node/InvokeAsyncCommand'

### ISSUE THREAD

**Issue by tedconn** [aws/aws-sdk-js-v3#180]
What's the right command for invoking a Lambda function - I was trying something like this:

```javascript
const { LambdaClient } = require('@aws-sdk/client-lambda-node/LambdaClient');
const { InvokeAsyncCommand } = require('@aws-sdk/client-lambda-node/InvokeAsyncCommand');

var lambda = new LambdaClient({region: 'eu-central-1'});
var params = {
  FunctionName: 'function:myFunctionName',
  DryRun: false,
  Publish: true,
  RevisionId: 'STRING_VALUE',
  ZipFile: data
};

const invokeAsyncCommand = new InvokeAsyncCommand(params);
lambda.send(invokeAsyncCommand).then(data => {
  log(chalk.green(data));
}).catch(error => {
  log(chalk.red(error));
});
```

But I am getting `Cannot find module '@aws-sdk/client-lambda-node/InvokeAsyncCommand'`

**Comment by chrisradek** [comment]
@tedconn 
You're very close, see this [readme](https://github.com/aws/aws-sdk-js-v3/tree/master/packages/client-lambda-node#import) for more examples.
The command should be accessible via "@aws-sdk/client-lambda-node/__commands__/InvokeAsyncCommand".

If you don't care about bundle sizes, you can also import the top-level `Lambda` class and use the client similar to how you would in the V2 SDK:
```typescript
const {Lambda} = require('@aws-sdk/client-lambda-node');
const client = new Lambda({region: 'eu-central-1'});
client.invokeAsync({/*params*/}
  .then((result) => {})
  .catch((error) => {}};
```

**Comment by tedconn** [comment]
Thanks! 

Looks like there's a doc issue because in the readme you link, the typescript import uses `/command/InvokeAsyncCommand` and the javascript example does not. I am in JavaScript. 

also I'm not super versed with TypeScript but seems to me like they should be the same : )

**Comment by srchase** [comment]
@tedconn 

Thanks for pointing that out!  We'll get that corrected.

**Comment by srchase** [comment]
Fixed in #182.  

If you build the clients from source now, the docs will have this change, otherwise the NPM version will get updated with the next release.
